test/bdd/fixtures/demo/docker-compose-edge-components.yml

```diff
@@ -170,6 +170,7 @@ services:
       - EDV_DATABASE_URL=${COUCHDB_USERNAME}:${COUCHDB_PASSWORD}@shared.couchdb:5984
       - EDV_DATABASE_PREFIX=edv
       - VIRTUAL_HOST=edv.trustbloc.local
+      - EDV_LOG_LEVEL=debug
```

Comment:
I have this here since it helps me for testing purposes. Shouldn't be used generally since it causes a ton of logging output.

Reply:
(removed now that the PR is ready)